spec/lib/rx/api/prescriptions_spec.rb

```diff
@@ -102,4 +102,32 @@
         .to raise_error(Common::Client::Errors::ClientResponse, 'Prescription is not Refillable')
     end
   end
+
+  context 'integration test for breakers' do
+    it 'is raises a breakers exception after 50% failure rate' do
+      now = Time.current
+      start_time = now - 120
+      Timecop.freeze(start_time)
+
+      stub_varx_request(:get, 'mhv-api/patient/v1/prescription/getactiverx', active_rxs, status_code: 200)
+      20.times do
+        client.get_active_rxs
+      end
+
+      stub_varx_request(:get, 'mhv-api/patient/v1/prescription/getactiverx', active_rxs, status_code: 500)
+      20.times do
+        begin
+          client.get_active_rxs
+        rescue Common::Client::Errors::ClientResponse
+          nil
+        end
+      end
+
+      expect { client.get_active_rxs }.to raise_exception(Breakers::OutageException)
```

Comment:
So this is awesome, but will this also be a serializable error class? What will we want to return to the api consumer? Maybe an Internal Server is satisfactory, but we could possibly have a more detailed error message that gets rendered. Either one of Common::Exceptions or something new.